src/components/SearchFilterList/SearchFilter.js

```diff
@@ -0,0 +1,119 @@
+import { useState, useMemo } from 'react'
+import { useResponsive } from 'hooks/useResponsive'
+import { Box, CheckBox, Heading, Text } from 'grommet'
+import { Button as sharedButton } from 'components/shared/Button'
+import { SearchBox } from 'components/shared/SearchBox'
+import { formatString } from 'helpers/formatString'
+import { scrollToId } from 'helpers/scrollToId'
+import styled, { css } from 'styled-components'
+
+const ToggleButton = styled(sharedButton)`
+  border-bottom: 1px solid transparent;
+  ${({ theme }) => css`
+    &:hover {
+      border-bottom: 1px solid ${theme.global.colors.brand};
+    }
+  `}
+`
+
+export const SearchFilter = ({ filterGroup, label }) => {
+  const { setResponsive } = useResponsive()
+  const options = useMemo(() => {
+    return Object.entries(filterGroup)
+  }, [filterGroup])
+  const [filteredResult, setfilteredResult] = useState(options)
+  const [open, setOpen] = useState(false)
+  const [userInput, setUserInput] = useState('')
+  const maxCount = 5
+  const filterLength = options.length
+
+  const filterOptions = (val) => {
+    setUserInput(val)
+
+    if (val.trim() !== '') {
+      setfilteredResult(() =>
+        options.filter((option) =>
+          formatString(option[0]).toLowerCase().startsWith(val.toLowerCase())
+        )
+      )
+    } else {
+      setfilteredResult(options)
+    }
+  }
+
+  const getOptionsToRender = () =>
+    open ? filteredResult : filteredResult.slice(0, maxCount)
+
+  return (
+    <>
+      <Heading
+        level={4}
+        margin={{ bottom: 'xsmall' }}
+        id={label.toLowerCase()}
+        size={setResponsive('h4_xsmall', 'medium')}
+      >
+        {label}
+      </Heading>
+
+      {filterLength > maxCount && (
+        <SearchBox
+          pad={{ bottom: 'xsmall' }}
+          placeholder={`Filter ${label}`}
+          value={userInput}
+          size="small"
+          changeHandler={(e) => filterOptions(e.target.value)}
+        />
+      )}
+
+      <Box
+        margin={{ top: 'xsmall' }}
+        animation={open ? { type: 'fadeIn', duration: 1000 } : {}}
+      >
+        {getOptionsToRender().map((option, i, arr) => (
+          <Box
+            key={option[0]}
+            margin={{ bottom: i !== arr.length - 1 ? 'xsmall' : '0' }}
+          >
+            <CheckBox
+              label={`${formatString(
+                option[0]
+              )} (${option[1].toLocaleString()})`}
+            />
+          </Box>
+        ))}
+      </Box>
```

Comment:
Another option here is to use `Grid` instead and set a `pad`.

```suggestion
      <Grid
        margin={{ top: 'xsmall' }}
        animation={open ? { type: 'fadeIn', duration: 1000 } : {}}
        pad={{ row: 'xsmall' }}
      >
        {getOptionsToRender().map((option) => (
          <Box key={option[0]}>
            <CheckBox
              label={`${formatString(
                option[0]
              )} (${option[1].toLocaleString()})`}
            />
          </Box>
        ))}
      </Grid>
```

Reply:
🗒️ Used `gap={{ row: 'xsmall' }}` instead of `pad`